### PR TITLE
Automatically generate `readthedocs` list of all plugins and their info

### PR DESCRIPTION
**Pull request by kvfairchild**
Populates `readthedocs` documentation (`plugins.rst`) with information about all available plugins and a project bibliography. Relies on information, e.g. BiBTeX, provided in plugin `__init__.py`. 

Adds the following dependencies:
* [`sphinxcontrib-bibtex`](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/index.html#)
* [`RstCloth`](https://rstcloth.readthedocs.io/en/latest/#rstcloth)

**Plugin info listing**
image

**Bibliography**
image

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -21,7 +21,7 @@ The `score` function is the primary entry point to score a model on a benchmark.
    :caption: Contents:
 
    modules/artificial_subject
-   brainscore_language/plugins
    modules/model_tutorial
    modules/benchmark_tutorial
    modules/api_reference
+   modules/plugins
```
Comment (mschrimpf) [inline docs/source/index.rst:27]:
```suggestion
   modules/plugins
   modules/api_reference
```

File: docs/source/modules/plugins.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
+
```
Comment (mschrimpf) [inline docs/source/modules/plugins.rst:1]: so will this file be auto-generated by the readthedocs build? I.e. won't be part of the repository once that happens?

**Review by mschrimpf — COMMENTED** [review]
Very cool. The type (data, benchmarks etc) and instance (Pereira2018.language, wikitext-2/text etc) headings look fairly similar on the page, but that's a sphinx issue.

Is there a way to also get the class documentation in there? E.g. the [`Futrell2018`](https://github.com/brain-score/language/blob/b03ea2c9ceef646c53a7caedcab9ac13df07354a/brainscore_language/benchmarks/futrell2018/__init__.py#L17-L26) benchmark has documentation. The [`_Pereira2018`](https://github.com/brain-score/language/blob/b03ea2c9ceef646c53a7caedcab9ac13df07354a/brainscore_language/benchmarks/pereira2018/__init__.py#L55-L70) base class also has documentation, I think there's a way to tell the calling functions to copy the documentation

File: docs/source/index.rst (hunk 1 above)
Comment (mschrimpf) [reply docs/source/index.rst:27]: API reference at the bottom

**Comment by mschrimpf** [comment]
@kvfairchild what's the status of this PR?

**Comment by kvfairchild** [comment]
> @kvfairchild what's the status of this PR?

Deprioritized behind plugin environments.

**Comment by mschrimpf** [comment]
Sorry, this PR was closed after combining commit history. @kvfairchild can you please open a new PR?

**Pull request closed without merging**